resq_tools/lib/screens/blattler_screen.dart

```diff
@@ -2,23 +2,81 @@ import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:resq_tools/blocs/blattler_cubit.dart';
 import 'package:resq_tools/utils/extensions.dart';
+import 'package:syncfusion_flutter_pdfviewer/pdfviewer.dart';
 
-class BlattlerScreen extends StatelessWidget {
+class BlattlerScreen extends StatefulWidget {
   const BlattlerScreen({super.key});
 
+  @override
+  State<BlattlerScreen> createState() => _BlattlerScreenState();
+}
+
+class _BlattlerScreenState extends State<BlattlerScreen> {
+
+  final PdfViewerController _pdfViewerController = PdfViewerController();
+  final TextEditingController _searchController = TextEditingController();
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: Text(context.l10n?.blattler_title ?? '')),
-      body: Center(
-        child: BlocBuilder<BlattlerCubit, BlattlerState>(
-          builder: (context, state) {
-            return state.isLoading
-                ? const CircularProgressIndicator()
-                : Text(state.dummyString ?? 'No data');
-          },
-        ),
+      appBar: AppBar(
+        title: Text(context.l10n?.blattler_title ?? 'Blattler'),
```

Comment:
Usually never null - replace it with empty string

```suggestion
        title: Text(context.l10n?.blattler_title ?? ''),
```